test/fixtures_pilot.js

```diff
@@ -64,6 +60,81 @@ module.exports = function() {
                     "data": {}
                 }
             }
-        }
+        },
+        post_outbound_message: function(params) {
+            // defaulting to this identity + address as its used in many
```

Comment:
This comment says that we're defaulting to a specific identity and address, but below it looks like we're not defaulting to a specific identity and address.